src/test/java/org/haffson/adventofcode/utils/FileReadersTest.java

```diff
@@ -1,18 +1,32 @@
 package org.haffson.adventofcode.utils;
 
+import org.junit.Assert;
```

Comment:
Note here that you are using the assertions from the `org.junit` package. This is perfectly fine, but most of the time we prefer to use AssertJ since it has many more powerful assertions. Maybe you can test them in a subsequent problem you solve.

Reply:
Okay, then I will switch to the AssertJ.